app/src/main/java/net/squanchy/schedule/view/TalkEventItemView.kt

```diff
@@ -0,0 +1,49 @@
+package net.squanchy.schedule.view
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.View
+import kotlinx.android.synthetic.main.item_schedule_event_talk.view.*
+import net.squanchy.R
+import net.squanchy.schedule.domain.view.Event
+import net.squanchy.support.lang.Optional
+import net.squanchy.support.widget.SpeakerView
+import org.joda.time.format.DateTimeFormat
+
+class TalkEventItemView @JvmOverloads constructor(
+        context: Context
+        , attrs: AttributeSet,
```

Comment:
The first comma should be at the end of the previous line

Reply:
9ee2e8016a801c66f3c21e6ed65642ce03cf1a2a